docs/flow/tutorials/scraping-the-agent.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/scraping-the-agent
+title: Scraping the Agent
```

Comment:
We need to be considerate of who our audience is here; newcomers probably won't know what "Scraping" means.

I think we should keep the old title: "Collecting and Prometheus metrics" so people understand what they're being taught, and then explain the scraping/remote_write terminology in the tutorial itself.